### ISSUE TITLE
Video to support accessibility standards

### ISSUE THREAD

**Issue by JoeButler7** [grommet/grommet#4176]
### Expected Behavior
In `Video-test.js` the `Video` component should have no accessibility violations and support a11yTitle

### Actual Behavior
The `Video` component does have accessibility violations due to a messing `aria-label`. When you try and use `a11yTitle` instead of `aria-label` there is still violations because `Video` does not support `a11yTitle`

### URL, screen shot, or Codepen exhibiting the issue
<img width="938" alt="Screen Shot 2020-06-11 at 4 46 49 PM" src="https://user-images.githubusercontent.com/49867133/84437562-34dd0200-ac03-11ea-80c2-18b4edf9a92d.png">
<img width="910" alt="Screen Shot 2020-06-11 at 4 46 14 PM" src="https://user-images.githubusercontent.com/49867133/84437578-3e666a00-ac03-11ea-8ea4-c3934db73e3c.png">

### Steps to Reproduce
Run `yarn Video-test`, add a11yTitle to component with accessibility violations and rerun `yarn test Video-test`
### Your Environment

- Grommet version: 2.13.0
- Browser Name and version: N/A
- Operating System and version (desktop or mobile): macOS catalina 10.15.4
